quantumdb-core/src/main/java/io/quantumdb/core/migration/operations/AlterColumnMigrator.java

```diff
@@ -50,8 +50,8 @@ public void migrate(Catalog catalog, RefLog refLog, Version version, AlterColumn
 			}
 		});
 
-		operation.getHintsToDrop().stream().forEach(column::dropHint);
-		operation.getHintsToAdd().stream().forEach(column::addHint);
+		operation.getHintsToDrop().forEach(column::dropHint);
+		operation.getHintsToAdd().forEach(column::addHint);
```

Comment:
.stream() does not guarantee it will be executed in order: https://www.baeldung.com/java-collection-stream-foreach

Reply:
True, but if it does a `.forEachOrdered()` can be used which does support this. The change itself is 👍 though